### PR TITLE
[Danger] re-enable proselint, add more danger rules

### LINKED ISSUE #333: Re-enable danger if we can make it useful
See #332 and https://github.com/SwiftWeekly/swiftweekly.github.io/pull/331#issuecomment-341521265.

Would be nice if danger-prose actually worked as advertised.

### PR DESCRIPTION
**Pull request by jessesquires**
close #333

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -1,6 +1,35 @@
-# Look for prose issues
-prose.lint_files
+# welcome first-time contributors
+welcome_message._test
```
Comment (BasThomas) [inline Dangerfile]: What does this do?

Comment (jessesquires) [reply Dangerfile]: supposed to be a "welcome message" to first time contributors if i can type it right 😆 

just pushed a fix

**Comment by jessesquires** [comment]
Going to open another PR stacked on this to test

**Comment by BasThomas** [comment]
You can do so in this PR :)

**Comment by BasThomas** [comment]
... and it seems to work

**Comment by jessesquires** [comment]
Ha. yeah pushed a fix 😄

**Comment by jessesquires** [comment]
Hrm. 🤔 I guess `prose.check_spelling` is broken. can't get it to work.

Oh well. not checking spelling seems to cut out a lot of the noise from before.

**Pull request merged**